oxo.py

```diff
@@ -1,26 +1,72 @@
 class OxoBoard:
```

Comment:
A lot of this code is very similar to that submitted by another student a few days before you. In general it's fine to work with your peers on these worksheets, however make sure you give them due credit in a comment in the code and/or a readme file, and ideally stick to sharing approaches to the problem rather than code itself